gatorgrouper/tests/test_read_student_file.py

```diff
@@ -1,7 +1,22 @@
 """Testing number of groups"""
+import pytest
+import csv
+#from coreapi import Client
 from utils import read_student_file
 
 
-def test_read_file_populates_csvdata_0():
-    """Checks that the read student file populates data from a CSV."""
-    assert read_student_file.read_student_file("gatorgrouper/tests/students.csv") != 0
+@pytest.fixture(scope='session')
+def file_name(tmpdir_factory):
```

Comment:
This is good but the fixture should have a more specific name -- perhaps `csv_file`? Otherwise, if there are other tests that want to use file fixtures, there might be conflicts in the future.

Reply:
I have changed to `generate_csv`, would that work? or `csv_name` makes more sense.

Reply:
It could probably be either `generated_csv` or something like that, but your current name is fine with with me.